product-service/create-product.js

```diff
@@ -0,0 +1,61 @@
+import {Client} from 'pg';
+
+const {PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD} = process.env,
+  dbOptions = {
+    host: PG_HOST || 'localhost', 
+    port: PG_PORT || 5432,
+    database: PG_DATABASE,
+    user: PG_USERNAME || 'postgres',
+    password: PG_PASSWORD,
+    ssl:{
+      rejectUnauthorized: false
+    },
+    connectionTimeoutMillis: 5000
+  };
+
+export const createProduct = async event => {
+    const client = new Client(dbOptions);
+
+    await client.connect();
```

Comment:
Try to reuse connection, instead of creating a new one for each Lambda invocation